pkg/datastore/helper.go

```diff
@@ -55,7 +55,7 @@ type DBConfig struct {
 	sslMode  string
 }
 
-func FromEnv(l *logrus.Entry, authorizer authorizer.Authorizer, instancer authorizer.Instancer) (d DataStore, err error) {
+func ConfigFromEnv(dbName string) DBConfig {
```

Comment:
Could you add header comments to this function and mention what the input parameter is for?